styles.css

```diff
@@ -325,88 +288,43 @@ h2 {
   border-radius: 8px;
 }
 
-.skill-js {
+.skill-ruby {
   display: flex;
   flex-direction: row;
   align-items: center;
   padding: 4px 12px;
-  width: 89px;
+  width: 110px;
   height: 24px;
   background: #ebebff;
   border-radius: 8px;
 }
 
-.skill-content {
-  position: relative;
-  color: #6070ff;
-}
-
-.button {
-  box-sizing: border-box;
+.skill-js {
   display: flex;
   flex-direction: row;
-  padding: 12px;
-  gap: 10px;
-  width: 130px;
-  height: 48px;
-  border: 1px solid #7f8cff;
-  background: #fff;
-  border-radius: 8px;
-  margin-top: 10px;
-}
-
-button:enabled {
-  display: flex;
-  justify-content: center;
   align-items: center;
-  padding: 12px;
-  gap: 10px;
-  left: 100px;
-  background: #fff;
-  border: 1px solid #7f8cff;
-  border-radius: 8px;
-}
-
-button:hover {
-  display: flex;
-  align-items: center;
-  padding: 12px;
-  gap: 10px;
-  width: 140px;
-  height: 48px;
-  background: #2230d2;
+  padding: 4px 12px;
+  width: 85px;
+  height: 24px;
+  background: #ebebff;
   border-radius: 8px;
 }
 
-.resume {
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  padding: 12px;
-  gap: 10px;
-  width: 162px;
-  height: 48px;
-  border: 1px solid #6070ff;
-  border-radius: 8px;
-  margin-top: 10px;
+.skill-content {
+  position: relative;
+  color: #6070ff;
 }
 
-.resume p {
-  color: #396df2;
-  width: 138px;
-  height: 24px;
+/* about-section */
+.about-section {
+  background-color: #fff;
 }
 
-h5 {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
-  display: flex;
-  justify-content: flex-start;
-  color: #000;
-  margin-bottom: 10px;
+.section-about {
+  flex-direction: column;
+  justify-content: space-around;
+  margin: 20px;
+  height: 300px;
 }
 
 .frame0 {
```

Comment:
- [ REQUIRED] As the previous reviewer requested, Kindly work on the margins of this section. please ensure that the margin of elements is uniform to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/194575774-cb1fcb98-2685-4a2b-8a75-3a317d92cf46.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/194575951-41eb7e11-a8f6-46f4-83c2-019b680056b7.png)

- [ REQUIRED] Kindly reduce the size of the background image on the contact section to match the Figma design as the previous reviewer requested

![image](https://user-images.githubusercontent.com/98453979/194576469-477bbf11-08fd-4f91-b6d7-857ff6024e84.png)




- [OPTIONAL ] In your about section, Kindly adjust the padding to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/194570847-a59697c2-e900-4224-aca2-26bb851b0d48.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/194571121-34cbc7b1-caa1-4d29-9c46-30bef32089c3.png)

- [OPTIONAL]  Kindly reposition your language cards to match the Figma design. They are supposed to be aligned horizontally not vertically. There should be two grey lines separating the frameworks and skills but I see you only have one.

![image](https://user-images.githubusercontent.com/98453979/194571939-2229acb9-618e-4839-835c-62c70183f7a9.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/194572113-a8ede0a5-cfec-457d-a1a5-4b8162cd4dc5.png)

- [OPTIONAL] Kindly increase the width of your inputs to match the Figma design and give them an equal width. After that, make sure the `get in touch` button is aligned to the center.

![image](https://user-images.githubusercontent.com/98453979/194573005-432d2d18-9333-4f64-a6d1-9048f1c023b0.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/194573194-ace44030-20bb-4feb-b341-4103befd4b4c.png)